modules/porous_flow/test/tests/chemistry/tests

```diff
@@ -1,183 +1,305 @@
 [Tests]
-  [./except1]
+  [except1]
     type = RunException
     input = except1.i
     expect_err = 'PorousFlowMassFractionAqueousEquilibriumChemistry: The number of secondary activity coefficients is 1 which must be equal to the number of secondary species \(2\)'
     threading = '!pthreads'
-  [../]
-  [./except2]
+    requirement = "PorousFlow shall produce an error if the incorrect number of activity coefficients is supplied to PorousFlowMassFractionAqueousEquilibriumChemistry."
```

Comment:
The system shall produce an error if the incorrect number of activity coefficients when forming mass fractions of solute species.

just a suggestion. Writing it this way (The system shall + no object names) is closer to how we do it at INL

Reply:
Will make this change